api/services/authentification.js

```diff
@@ -0,0 +1,46 @@
+const Crypto  = require('crypto')
+const Storage = require('../../storage/storage')
+
+const generateToken = () => Crypto.randomBytes(16).toString("hex"),
+
```

Comment:
it is token length

Reply:
Aha, okay, thank you :) Then it would be nice to accumulate such magic numbers to some constants:
const TOKEN_LENGTH=16.
And then using the constant would make the code more obvious :)